src/ApparentHorizons/ChangeCenterOfStrahlkorper.cpp

```diff
@@ -0,0 +1,167 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "ApparentHorizons/ChangeCenterOfStrahlkorper.hpp"
+
+#include <cmath>
+
+#include "ApparentHorizons/Strahlkorper.hpp"
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "NumericalAlgorithms/RootFinding/TOMS748.hpp"
+#include "Utilities/ConstantExpressions.hpp"
+#include "Utilities/GenerateInstantiations.hpp"
+#include "Utilities/Gsl.hpp"
+#include "Utilities/MakeWithValue.hpp"
+
+namespace change_expansion_center_of_strahlkorper_detail {
+
+// Get the r_hat vector of the Strahlkorper, which defines the
+// collocation points.
+template <typename Frame>
+tnsr::i<DataVector, 3, Frame> get_rhat(
+    const Strahlkorper<Frame>& strahlkorper) noexcept {
+  const auto new_theta_phi = strahlkorper.ylm_spherepack().theta_phi_points();
+  auto r_hat_l =
+      make_with_value<tnsr::i<DataVector, 3, Frame>>(new_theta_phi[0], 0.0);
+  const DataVector sin_theta = sin(new_theta_phi[0]);
+  get<0>(r_hat_l) = sin_theta * cos(new_theta_phi[1]);
+  get<1>(r_hat_l) = sin_theta * sin(new_theta_phi[1]);
+  get<2>(r_hat_l) = cos(new_theta_phi[0]);
+  return r_hat_l;
```

Comment:
Why not:
```cpp
return {sin_theta * cos(new_theta_phi[1]), sin_theta * sin(new_theta_phi[1]), cos(new_theta_phi[0])};
```
You may need more braces but I think this should work. You are able to construct vectors by their components, but I'm unsure of the expression templates will give you grief.

Reply:
I didn't know you can do this!  There are similar things in my other PRs that I can change in the same way.
  Yes, there was some grief because the constructor of Tensor that takes a std::array is `explicit`, but that wasn't too hard to get around.